command-expansion-server/build.gradle

```diff
@@ -12,6 +12,12 @@ task processResources {
   // Currently does nothing, exists as a hook for `distributeSql`
 }
 
+task assemble(type: NpmTask) {
+  dependsOn processResources
+  dependsOn npmInstall
+  args = ['run', 'build']
+}
+
```

Comment:
Since `build` and `assemble` do the same things now I'd suggest making `build` defined as:
```groovy
task build {
  dependsOn assemble
}
```